configs/fourdag/README.md

```diff
@@ -77,7 +77,7 @@ The 2D keypoints and pafs data we use is generated by openpose, and you can down
 
 | Config | Actor 0 | Actor 1 | Actor 2 | Average | Download |
```

Comment:
In addition to the detailed PCP values mentioned in the paper, shall we also report the results of other metrics selected in `pick_dict` if they are ready? For MvP, MPJPE, PA-MPJPE, PCK@50, and PCK@100 are reported additionally for Shelf and Campus.